_pages/plugins/ds4h-image-alignment.md

```diff
@@ -7,11 +7,11 @@ name: "DS4H Image Alignment"
 release-date: "2019-08-13"
 initial-release-date: "2019-06-08"
 dev-status: "stable"
-team-founders: [Stefano Belli, Antonella Carbonaro, Filippo Piccinini]
-team-maintainer: Stefano Belli
+team-founders: [Antonella Carbonaro, Filippo Piccinini, UniBoDS4H, Stefano Belli, Marco Edoardo Duma]
+team-maintainer: UniBoDS4H
```

Comment:
You should put an `@` symbol before any user IDs like `UniBoDS4H`. And create the corresponding page under the `_pages/people` directory, or it won't be able to pull the metadata, because that's where it gets the name from.

Reply:
But since `UniBoDS4H` is an org, not a GitHub user, actually don't do this. You can do it for `Edodums`, though, if you like.

Reply:
Actually UniBoDS4H is a "user" ( https://github.com/UniBoDS4H ). Anyway thanks :) I'll create the page within today.
Anyway I can't do it with my personal profile because I've already completed my work and I'll most likely won't return to do anything else in this project ( It was related to my bachelor's degree thesis ).